utils/scheduler.py

```diff
@@ -0,0 +1,29 @@
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from utils.core import get_random_quote
+import logging
+from logging.config import fileConfig
+
+fileConfig('../utils/logging_config.ini')
+logger = logging.getLogger('discord')
+
+
+def init_message_scheduler(config, client):
+
+    jobs = config.get("SCHEDULES")
+    if not jobs or not jobs[0]["MESSAGES"]:
+        "No jobs found in provided config."
```

Comment:
This is a string where I believe you intended to place a comment. e.g. """No jobs found in provided config."""